tox.ini

```diff
@@ -3,8 +3,9 @@ skipsdist = True
 usedevelop = True
 
 envlist =
-    python{3.8,3.9,3.10}-django{3.2,4.1}-wagtail{4.1,4.2,5.0,5.1,main}-{sqlite,postgres}
-    python{3.11}-django{4.1,4.2}-wagtail{5.0,5.1,main}-{sqlite,postgres}
```

Comment:
I had to also drop the `wagtailmain` tests because the `modeladmin` is already deprecated on the newest wagtail version.

Making a note here to put the `wagtailmain` tests back for Wagtail 6.0+ upgrades.